pywbemtools/pywbemcli/_cmd_statistics.py

```diff
@@ -0,0 +1,443 @@
+           subcommand_metavar=SUBCMD_HELP_TXT)
+@add_options(help_option)
+def statistics_group():
+    """
+    Command group for WBEM server statistics.
+
+    This command group defines commands to control the display of statistical
+    data managed by pywbemcli based on the ``--timestats`` general
+    option which enables the gathering and display of statistics in pywbemcli.
+
+    Since there may also be capabilities to manage statistics in WBEM servers,
+    these commands provide for a) enabling statistics gathering in the WBEM
+    server and b) displaying the information gathered.
+
+    One component of the pywbemcli client side statistics depends on the state
+    of statistics gathering enabled in the WBEM server, reporting the server
+    response time. This appears as the column "Server Times" in the report if
+    statistics gathering in the server is enabled.
+
+    Since gathering of statistics in a WBEM server may depend on a server
+    setting, pywbemcli provides commands to enable and disable the statistics
+    gathering in the server by setting the CIM_ObjectManager
+    "GatherStatisticalData" property.
+
+    In addition to the command-specific options shown in this help text, the
+    general options (see 'pywbemcli --help') can also be specified before the
+    'statistics' keyword.
+    """
+    pass  # pylint: disable=unnecessary-pass
+
+
+@statistics_group.command('reset', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_reset(context):
+    """
+    Reset the counts in the client managed statistics.
+
+    This command resets the counts in the statistics kept by pywbemcli for the
+    current connection including the statistics on the server response times
+    received from the WBEM server in the ``WBEMServerResponseTime`` header .
+
+    It does not reset statistics managed by the WBEM server..
+    """
+    # pylint: disable=too-many-function-args
+    context.execute_cmd(lambda: cmd_statistics_reset(context))
+
+
+@statistics_group.command('server-on', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_server_on(context):
+    """
+    Enable statistics on current server.
+
+    This command  activates the gathering of server statistics in the WBEM
+    server and the return of server response times for inclusion in
+    pywbemcli maintained statistics. The gathering of server statistics and the
+    returning of server response times may not be implemented by all WBEM
+    servers.
+
+    This may fail if the server does not manage statistics or does not allow
+    a client to modify the state of statistics gathering.
+
+    WBEM server statistics gathering state does not impact pywbemcli client
+    statistics gathering other than whether the 'Server Times` row is included
```

Comment:
t is a column, and it is labeled "Server Time".

Reply:
DONE